app/controllers/result_filters/location_controller.rb

```diff
@@ -11,30 +11,27 @@ def start
     end
 
     def create
+      # if searching for specific provider go to results page
```

Comment:
This comment needed? Also looks like it's going to provider_path not results_path

Reply:
I thought it might be handy to know that the provider_path also does a redirect depending on whether the provider name exists and only returns 1 record else it redirects the user to a provider suggestion page for them to select a specific one.

Reply:
IMO changing the method `provider_option_selected?` would be much clearer than having a comment - I had a similar response to toby in that I read this comment and saw the redirect and felt like this was an out of place/out of date comment. Would be worth improving the code if you think it's not clear